src/GlobalStyle.js

```diff
@@ -7,9 +7,24 @@ export default createGlobalStyle`
 
   body {
     margin: 0;
-    font-family: sans-serif;
+    font-family: 'Inter', sans-serif;
     line-height: 1.5;
     font-size: 112.5%;
     color: #585858;
```

Comment:
You could safe this in variables (if you haven't planned this allready for the next stories)